source/javascripts/controllers/_TriggersController.js.erb

```diff
@@ -190,10 +206,6 @@
 			};
 
 			viewModel.triggerIsInvalid = function() {
-				if (viewModel.editedTrigger.workflowID()[0] == "_") {
-					return true;
-				}
```

Comment:
This should still be checked, but with first doing a check if the workflow's ID is defined.

Reply:
Well I made them unselectable from the dropdown, but leaving an extra check shouldn't hurt